### ISSUE TITLE
pyodide option not work in jupyter lite serve

### ISSUE THREAD

**Issue by liudonghua123** [jupyterlite/pyodide-kernel#43]
## Description

I tried to execute `jupyter lite serve --pyodide=pyodide --mathjax-dir=MathJax-2.7.5` after `jupyter lite init--pyodide=pyodide --mathjax-dir=MathJax-2.7.5`. The init command worked, but serve command failed with the following errors.

```
......
.  build:pyodide:copy:pyodide
TaskError - taskid:build:pyodide:copy:pyodide
PythonAction Error
Traceback (most recent call last):
  File "C:\Users\Liu.D.H\AppData\Local\Programs\Python\Python310\lib\site-packages\doit\action.py", line 437, in execute
    returned_value = self.py_callable(*self.args, **kwargs)
  File "C:\Users\Liu.D.H\AppData\Local\Programs\Python\Python310\lib\site-packages\jupyterlite\addons\base.py", line 65, in copy_one
    shutil.copy2(src, dest)
  File "C:\Users\Liu.D.H\AppData\Local\Programs\Python\Python310\lib\shutil.py", line 434, in copy2
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "C:\Users\Liu.D.H\AppData\Local\Programs\Python\Python310\lib\shutil.py", line 254, in copyfile
    with open(src, 'rb') as fsrc:
FileNotFoundError: [Errno 2] No such file or directory: 'D:\\code\\python\\jupyterlite\\test\\.cache\\pyodide\\pyodide\\pyodide'
```

## Reproduce

1. unzip pyodide and mathjax to the cwd
2. jupyter lite int --pyodide=pyodide --mathjax-dir=MathJax-2.7.5
3. jupyter lite serve --pyodide=pyodide --mathjax-dir=MathJax-2.7.5

## Expected behavior

The `serve` command should work.

## Context

- JupyterLite version: 0.1.0b3
- Operating System and version: Microsoft Windows [Version 10.0.22000.556]
- Browser and version: Chrome 99

**Comment by tomjakubowski** [comment]
@lesteve I was seeing an error like this when using a directory with `--pyodide` too:

```
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/test/.cache/pyodide/pyodide/pyodide'
```

I found that the problem in my case was directory structure. I had the pyodide's dist in a folder like this:

```
foo-pyodide/pyodide.js
foo-pyodide/pyodide.asm.js
...
```

and was building with `jupyter lite build --pyodide foo-pyodide/`.

I made a new directory `foo-pyodide/pyodide/`, moved the pyodide dist in there, and then again passed `--pyodide foo-pyodide/`. This worked as expected.

In other words, I think `--pyodide` expects to be pointed to a directory with a subdirectory called `pyodide` containing the files from the dist, not to the directory containing the dist itself.

NB: I saw the same problem when using a .tar.bz2 with the wrong directory structure too.
